lib/sycamore/sycamore/utils/cache.py

```diff
@@ -16,10 +18,24 @@ def set(self, hash_key: str, hash_value):
         pass
 
     @staticmethod
-    def get_hash_key(data: bytes) -> str:
+    def get_hash_bytes(data: bytes) -> hashlib.sha256:
         hash_sha256 = hashlib.sha256()
         hash_sha256.update(data)
-        return hash_sha256.hexdigest()
+        return hash_sha256
+
+    @staticmethod
+    def get_hash_file(file_path: str) -> hashlib.sha256:
```

Comment:
This function may be even more useful if you can pass in the hash context to be used.